taskcluster/.build.yml

```diff
@@ -9,9 +9,11 @@ build:
       true
   git_lfs:
     - >
-      export PATH=/home/build-user/bin:$PATH &&
+      export PATH=${system.homedir.linux}/bin:$PATH &&
       wget https://github.com/git-lfs/git-lfs/releases/download/v2.3.1/git-lfs-linux-amd64-2.3.1.tar.gz -O - | tar -C /tmp -zxf - &&
-      PREFIX=/home/build-user/ /tmp/git-lfs-2.3.1/install.sh
+      PREFIX=${system.homedir.linux}/ /tmp/git-lfs-2.3.1/install.sh
+  scripts:
+    build: 'tc-build.sh'
+    package: 'tc-package.sh'
```

Comment:
Kind of awkward to have these non-existent files here. Do they *have* to be defined here?

Reply:
They were left here to ensure to fail, but I can now remove that.